src/gui/guiFormSpecMenu.cpp

```diff
@@ -129,6 +131,28 @@ GUIFormSpecMenu::~GUIFormSpecMenu()
 	delete m_text_dst;
 }
 
+void GUIFormSpecMenu::create(GUIFormSpecMenu **cur_formspec, Client *client,
+	JoystickController *joystick, IFormSource *fs_src, TextDest *txt_dest)
```

Comment:
Does not belong to moving but as you're touching these lines: could you please use `GUIFormSpecMenu *&cur_formspec`?
-> `cur_formspec->doPause = false` etc.